microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -6,19 +6,79 @@ ms.author: msedgedevrel
+To publish an extension, you first use Partner Center, and then optionally you can use the REST API to update the extension (or use Partner Center instead).
 
-The Microsoft Edge Add-ons API provides a set of REST endpoints for programmatically publishing updates to add-ons submitted to the Microsoft Edge Add-ons store.  You can use these REST endpoints to automate the process of uploading and publishing new versions of your add-ons to the Microsoft Edge Add-ons store.  You use the **Publish API** page at Partner Center to enable the API and begin working with these API endpoints.
-
-<!-- To publish an extension, you first use Partner Center, and then optionally you can use the REST API to update the extension (or use Partner Center instead). -->
+The Microsoft Edge Add-ons API provides a set of REST endpoints for programmatically publishing updates to add-ons submitted to the Microsoft Edge Add-ons store.  You can use these REST endpoints to automate the process of uploading and publishing new versions of your add-ons to the Microsoft Edge Add-ons store.  You use the **Publish API** page at Partner Center to first enable the API and then begin using these API endpoints.
 
 To submit suggestions and feedback, enter an [Issue about the Add-ons API](https://github.com/MicrosoftDocs/edge-developer/issues/new?title=[Add-ons%20API]).
 
 
+<!-- ------------------------------ -->
+#### Versions of the Update REST API
+
+As of September 6, 2024, both v1.1 and v1 of this Update REST API are supported.  Later, v1 will no longer be supported.  The date for ending v1 support is not yet finalized.
+
+
+<!-- ------------------------------ -->
+#### Components used by the Update REST API
+
+##### [v1.1](#tab/v1-1)
+
+v1.1 uses API keys.  The UI in Partner Center covers API keys, as described below.
+
+v1.1 doesn't require sending an Access Token URL.
+
+The REST API endpoints use specific request headers for v1.1, as described in [REST API Reference for Microsoft Edge Add-ons](./addons-api-reference.md).
+
+##### [v1](#tab/v1)
+
+v1 uses Secrets.  The UI in Partner Center covers secrets, as described below.
+
+v1 requires sending an Access Token URL.  The UI in Partner Center covers access tokens, as described below.
+
+The REST API endpoints use specific request headers for v1, as described in [REST API Reference for Microsoft Edge Add-ons](./addons-api-reference.md).
+
+---
+
+
+<!-- ---------------------------------- -->
+#### Overview of using the Update REST API
+
+##### [v1.1](#tab/v1-1)
+
+To use v1.1 of the Update REST API:
+
+1. At your [Partner Center developer dashboard](https://partner.microsoft.com/dashboard/microsoftedge/public/login?ref=dd), opt-in to the API key management experience.
+
+1. The ClientId and API Keys are automatically re-generated, by backend services.<!-- todo: every time you use any endpoint of the REST API?  what action causes re-generation of ClientId and API Keys? -->
```

Comment:
what action triggers re-gen of ClientID & API keys?  when doing something at Partner Center, or using an API endpoint?
step might need rewording, given that this section is Overview of steps.

Reply:
closing Outdated comments.  added todo notes in .md; will create fresh Comment at each todo note